stock_picking_send_by_mail/models/stock_picking.py

```diff
@@ -6,30 +6,28 @@
 
 
 class StockPicking(models.Model):
-    _inherit = "stock.picking"
+    _inherit = 'stock.picking'
 
     def action_picking_send(self):
         self.ensure_one()
-        template = self.env.ref(
-            "stock_picking_send_by_mail.email_template_stock_picking", False,
-        )
-        compose_form = self.env.ref("mail.email_compose_message_wizard_form", False,)
+        template = self.env.ref('stock_picking_send_by_mail.email_template_stock_picking', False)
```

Comment:
Don't do format changes unless necessary. 
OCA has just 79 characters, not 115 like us